infra/k8s/backup.yaml

```diff
@@ -26,11 +26,11 @@ spec:
         spec:
           containers:
           - name: backup
-            image: gcr.io/clusterfuzz-images/base:vguidi_py311_testing_v1
+            image: gcr.io/clusterfuzz-images/base:091c6c2-202409251610
```

Comment:
Are we pinning these images but running the bots on latest?

Reply:
Bots will be pinned as well, a CL will show up internally soon

Reply:
OK!